azure-iot-device/azure/iot/device/constant.py

```diff
@@ -11,3 +11,4 @@
 IOTHUB_API_VERSION = "2018-06-30"
 PROVISIONING_API_VERSION = "2019-03-31"
 SECURITY_MESSAGE_INTERFACE_ID = "urn:azureiot:Security:SecurityAgent:1"
+SIZE_LIMIT = 262144
```

Comment:
>SIZE_LIMIT [](start = 0, length = 10)

size of what?  I suggest adding what size this limits.  (e.g. TELEMETRY_MESSAGE_SIZE_LIMIT)